File: algorithms/Array/Pangram/Pangram.py
x = input("Enter a string\n")
my_list = [0]*26
for letters in x :
    if letters == " ":
        continue
    elif ord(letters)>=65 and ord(letters)<=90:
        my_list[ord(letters)-ord('A')]+=1
    else:
        my_list[ord(letters)-ord('a')]+=1
if my_list.count(0) == 0:
    print("Its a Pangram\n")
else:
    print("Not a Pangram\n")

File: algorithms/Array/Pangram/Pangram.cpp
#include<bits/stdc++.h>
#define lli long long int
using namespace std;
int main()
{
    int i,j,count=0,l;
    int a[26]={0};
    char s[100] ;
    cin.getline(s,sizeof(s));
    l = strlen(s);
    for(i=0;i<l;i++)
    {
        if(s[i]>=65 && s[i]<= 90)
            a[s[i]-'A']++;
        else 
        a[s[i]-'a']++;    
    }
    for(i=0;i<26;i++)
        if(a[i]!=0)
            count++;
    if(count==26)
        cout << "Its a Pangram\n";
    else 
        cout << "Its not a Pangram\n";

    return 0;
}
